dashboard/app/discussion.go

```diff
@@ -88,16 +89,24 @@ func mergeDiscussion(c context.Context, update *dashapi.Discussion) error {
 		if err != nil {
 			return fmt.Errorf("failed to put Discussion: %w", err)
 		}
-		// Update individual bug statistics.
-		for _, key := range d.BugKeys {
-			err := mergeDiscussionSummary(c, key, d.Source, diff)
-			if err != nil {
-				return fmt.Errorf("failed to put update summary for %s: %w", key, err)
-			}
-		}
 		return nil
 	}
-	return db.RunInTransaction(c, tx, &db.TransactionOptions{Attempts: 15, XG: true})
+	err = db.RunInTransaction(c, tx, &db.TransactionOptions{Attempts: 15, XG: true})
+	if err != nil {
+		return err
+	}
+	// Update individual bug statistics.
+	// We have to do it outside of the main transaction, as we might hit the "operating on
+	// too many entity groups in a single transaction." error.
+	for _, key := range d.BugKeys {
```

Comment:
These are also probably the monthly reports that we want to somehow ignore.
What's the datastore limit on updated groups? It's always good to have some reasonable upper bounds on such things.

Reply:
AFAIK These were mostly stable backport patch series that could mention up to 30-40 fixed syzbot bugs. The limit was around 20.